diff --git a/install.js b/install.js
index 540e1e68a561fbd0b3af0438b0465e984a7d5b52..993438448286aabc751228632bb87f3138e9b273 100755
--- a/install.js
+++ b/install.js
@@ -38,6 +38,11 @@ if (
   }
 }
 
+process.env.ELECTRON_CUSTOM_DIR = "v{{ version }}-patched-2";
+process.env.ELECTRON_MIRROR = "https://github.com/thegu5/electron/releases/download/";
+process.env.ELECTRON_NIGHTLY_MIRROR = "https://github.com/thegu5/electron/releases/download/";
+process.env.electron_use_remote_checksums = "true"
+
 // downloads if not cached
 downloadArtifact({
   version,
